segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperConfig.java

```diff
@@ -39,6 +39,9 @@
     public static final Property<Integer> BK_LEDGER_MAX_SIZE = Property.named("bkLedgerMaxSize", 1024 * 1024 * 1024);
     public static final Property<String> BK_PASSWORD = Property.named("bkPass", "");
     public static final Property<String> BK_LEDGER_PATH = Property.named("bkLedgerPath", "");
+    public static final Property<Boolean> BK_TLS_ENABLED = Property.named("tlsEnabled", false);
+    public static final Property<String> BK_TLS_TRUSTSTORE = Property.named("tlsTrustStore", "../../../config/client.truststore.jks");
```

Comment:
1. `tlsTrustStorePath`, since this looks like a path.
2. These relative paths won't exist in the deployment. Chances are we'll have a different deployment structure than in our dev environments.

Reply:
Fixed.